src/EditorFeatures/CSharpTest/Completion/CompletionProviders/XmlDocumentationCommentCompletionProviderTests.cs

```diff
@@ -1135,5 +1234,99 @@ static void Foo(string str)
 {
 }", "param name=\"str\"");
         }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.Completion)]
+        public async Task ListTypes()
+        {
+            await VerifyItemsExistAsync(@"
+/// <summary>
+/// <list type=""$$""
+/// </summary>
+static void Foo()
+{
+}
+", "bullet", "number", "table");
+        }
+
+        [WorkItem(11490, "https://github.com/dotnet/roslyn/issues/11490")]
+        [Fact, Trait(Traits.Feature, Traits.Features.Completion)]
+        public async Task SeeLangwordValues()
+        {
+            await VerifyItemsExistAsync(@"
+/// <summary>
+/// <see langword=""$$""
+/// </summary>
+static void Foo()
+{
+}
+", "await", "class");
+        }
+
+        [WorkItem(11489, "https://github.com/dotnet/roslyn/issues/11489")]
+        [Fact, Trait(Traits.Feature, Traits.Features.Completion)]
+        public async Task SeeAttributesOnSpace()
+        {
+            var text = @"
+/// <summary>
+/// <see $$
+/// </summary>
+static void Foo()
+{
+}";
+            await VerifyItemExistsAsync(text, "langword", usePreviousCharAsTrigger: true);
+        }
+
+        [WorkItem(11489, "https://github.com/dotnet/roslyn/issues/11489")]
+        [Fact, Trait(Traits.Feature, Traits.Features.Completion)]
+        public async Task SeeLangwordValuesOnQuote()
+        {
+            var text = @"
+/// <summary>
+/// <see langword=""$$
+/// </summary>
+static void Foo()
+{
+}";
+            await VerifyItemExistsAsync(text, "await", usePreviousCharAsTrigger: true);
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.Completion)]
+        public async Task AttributeNameAfterElementName()
+        {
+            var text = @"
+class C
+{
+    /// <exception $$
+    void Foo() { }
+}
+";
+            await VerifyItemExistsAsync(text, "cref");
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.Completion)]
+        public async Task PartiallyTypedAttributeName()
+        {
+            var text = @"
+class C
+{
+    /// <exception c$$
+    void Foo() { }
+}
+";
+            await VerifyItemExistsAsync(text, "cref");
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.Completion)]
+        public async Task AttributeNameAfterAttribute()
+        {
+            var text = @"
+class C
+{
+    /// <exception name="""" $$
```

Comment:
is 'name' a valid 'exception' attribute?  If so, do we offer it in intellisense?  If so, can we add a test for that?